expirationd.lua

```diff
@@ -327,48 +390,202 @@ end
---                                supported index types are TREE and HASH, using other types will result in an error.
---     * iterator_type         -- Type of the iterator to use, as string or box.index constant,
---                                for example, 'EQ' or box.index.EQ; default is box.index.ALL;
---                                see https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_index/pairs/.
---     * start_key             -- Start iterating from the tuple with this index value;
---                                or when iterator is 'EQ', iterate over tuples with this index value;
---                                must be a value of the same data type as the index field or fields,
---                                or a function which returns such value;
---                                if omitted or nil, all tuples will be checked.
---     * tuples_per_iteration  -- Number of tuples to check in one batch (iteration); default is 1024.
---     * atomic_iteration      -- Boolean, false (default) to process each tuple as a single transaction;
---                                true to process tuples from each batch in a single transaction.
---     * process_while         -- Function to call before checking each tuple;
---                                if it returns false, the task will stop until next full scan.
---     * iterate_with          -- Function which returns an iterator object which provides tuples to check,
---                                considering the start_key, process_while and other options.
---                                There's a default function which can be overriden with this parameter.
---     * on_full_scan_start    -- Function to call before starting a full scan iteration.
---     * on_full_scan_complete -- Function to call after completing a full scan iteration.
---     * on_full_scan_success  -- Function to call after successfully completing a full scan iteration.
---     * on_full_scan_error    -- Function to call after terminating a full scan due to an error.
---     * args                  -- Passed to is_tuple_expired and
---                                process_expired_tuple() as additional context.
---     * full_scan_time        -- Time required for a full index scan (in seconds).
---     * iteration_delay       -- Max sleep time between iterations (in seconds).
---     * full_scan_delay       -- Sleep time between full scans (in seconds).
---     * force                 -- Run task even on replica.
---  }
+--- Run a scheduled task to check and process (expire) tuples in a given space.
+--
+-- How expirationd works in general:
+--
+-- 1. Process min(`space_length`, `tuples_per_iteration`) tuples at once.
+--
+-- 2. Sleep `tuples_per_iteration` × `full_scan_time` / `space_length` (but not
+--    beyond 1 second).
+--
+-- 3. Repeat 1-2 until the whole space will be traversed.
+--
+-- 4. Sleep 1 second.
+--
+-- 5. Repeat 1-4.
```

Comment:
1. Looks relevant to #38.
2. It is rendered as unnumbered list on https://tarantool.github.io/expirationd/, but when I generated it locally, the list is numbered. I guess the GitHub Pages webpage contains not a last state.

Reply:
2. Oleg already spotted it and I haven't update HTML version after fix.
Pushed latest version to https://tarantool.github.io/expirationd/.